### ISSUE TITLE
Add parity file support

### ISSUE THREAD

**Issue by gyurix** [javi11/usenet-drive#82]
Please add support for creating a configurable percentage of parity files to ensure Usenet won't magically lose our uploaded files so easily.

**Comment by javi11** [comment]
Will look at it in the future, but if this is implemented should be somehow an async process manually triggered by the admin panel since parity check can be an extensive task that can take time. For the moment, if you upload the file encrypted it, the files are available for a long time according to my experience.

**Comment by nicpaesk** [comment]
According to [this](https://www.reddit.com/r/usenet/comments/14hw7bt/comment/jph7at5/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) post, the main reason corruption happens is due to files moving among providers. Thus, if you read your files in the same or have multiple providers, it shouldn't be such an issue.
I do think it's a good idea, but something to maybe only if the file is detected as bad or missing by Usenet Drive?

**Comment by javi11** [comment]
Hey, the tool will mark your file as corrupted if that happens but still a par file will be nice to have to autorepare damaged files

**Comment by gyurix** [comment]
Yes, we need the files to not corrupt, but be reliably stored on Usenet. And par's can help with increasing that reliability factor.
